src/main/java/sx/blah/discord/handle/obj/IGuild.java

```diff
@@ -605,6 +605,15 @@
 	IEmoji getEmojiByName(String name);
 
 	/**
+	 * This adds a custom emoji to the guild.
+	 *
+	 * @param name The name, <b>without colons</b>.
+	 * @param image The BufferedImage of the Emoji.
+	 * @return The emoji.
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1782682/25064718/6770ac0c-21ce-11e7-972c-0c03227f67dc.png)
Going to go ahead and commit the changes to the jdoc